vagrant/provisioning/roles/alfresco7/tasks/main.yml

```diff
@@ -274,7 +274,7 @@
       messaging.broker.username={{ activemq_user | default('guest', true) }}
       messaging.broker.password={{ activemq_password | default('guest', true) }}
       index.subsystem.name=solr6
-      solr.secureComms=none
+      solr.secureComms=https
```

Comment:
Does Alfresco-to-Solr communication work with this change?  More changes may be needed to enable TLS between Alfresco Content Services and Alfresco Search Services.

Reply:
For local arkcase installment, yes. That is why i put this PR in WIP, so this kind of thoughts be shared in order to make adjustment to the code.